### ISSUE TITLE
SwiftClient.create is not a function

### ISSUE THREAD

**Issue by dfaustryjak** [stefanodefaria/swift-client#1]
const SwiftClient = require('openstack-swift-client');
let client = SwiftClient.create('https://orbit.brightbox.com/v1/acc-xxx', 'cli-xxx', 'my-password');

Can you give me example how can i usage it :) ?

**Comment by stefanodefaria** [comment]
I'm sorry, documentation hasn't been updated.
The correct way to use it is:

```js
const SwiftClient = require('openstack-swift-client');
let client = new SwiftClient('https://orbit.brightbox.com/v1/acc-xxx', 'cli-xxx', 'my-password');
```

**Comment by stefanodefaria** [comment]
Fixed documentation in 7f4e845bf541721790c275c9f3ff85818541b9a8.
